File: Backend/ArcadeX.Persistence/Entities/ReviewComment.cs
namespace ArcadeX.Persistence.Entities;

public class ReviewComment
{
    public int Id { get; set; }
    public Guid ReviewId { get; set; }
    public Guid UserId { get; set; }
    public string? Comment { get; set; }
    public DateTime CreatedAt { get; set; }
    public Review Review { get; set; } = null!;
    public User User { get; set; } = null!;
    
}

File: Frontend/src/app/feactures/interfaces/private/ReviewComment.ts
export interface IReviewComment {
  id: number;
  reviewId: string;
  userId?: string;
  comment: string;
  createdAt?: string;
}

export interface ICreateReviewComment {
  reviewId: string;
  comment: string;
}

export interface IUpdateReviewComment {
  comment: string;
}
